scene/main/node.h

```diff
@@ -94,7 +94,7 @@ class Node : public Object {
 
 	// This Data struct is to avoid namespace pollution in derived classes.
 	struct Data {
-		String scene_file_path;
+		String scene_file_path = "";
```

Comment:
String and StringName are default initialized, so these changes are unnecessary.